tools/generator/templates/rules/wscript.jinja2

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
I don't think we need a shebang actually, it's not supposed to be a main

Reply:
You are right we don’t need it. It kept it because we use it everywhere.

It does have a use though: it allows editors to know that this file is written in python. Without it there is no syntax highlighting and such. So it might be a good idea to use it.